ledger/lruaccts_test.go

```diff
@@ -195,3 +196,31 @@ func TestLRUAccountsOmittedPendingWrites(t *testing.T) {
 		require.Equal(t, persistedAccountData{}, acct)
 	}
 }
+
+func BenchmarkLRUAccountsWrite(b *testing.B) {
+	accounts := make([]persistedAccountData, 5000)
+	for i := range accounts {
+		digest := crypto.Hash([]byte{byte(i)})
+		microAlgos := binary.BigEndian.Uint64(digest.MarshalMsg(nil))
+
+		accounts[i] = persistedAccountData{
+			addr:        basics.Address(digest),
+			round:       basics.Round(i),
+			rowid:       int64(i),
+			accountData: basics.AccountData{MicroAlgos: basics.MicroAlgos{Raw: microAlgos}},
+		}
+	}
+
+	baseAccts := make([]lruAccounts, b.N)
+	for i := 0; i < b.N; i++ {
+		baseAccts[i].init(logging.TestingLog(b), 10, 5)
```

Comment:
In order to get realistic timings, you'll need to fill this data structure with dummy data.
Testing the performance again an empty datastructure is less than likly to represent a typical load.

Reply:
I see. 
i've changed the coed to generate 5k accounts for the benchmark, and made certain to fill the `baseAccts` with 95k different `persistedAccountData` before running the benchmark.